### PR TITLE
1784 update ansys multi core license restrictions

### PR DESCRIPTION
**Pull request by Carldkennedy**

### CODE REVIEW

**Comment by jkwmoore** [comment]
Can this be edited so each sub-software page using the controlled increment (anshpc) also has the warning:

`
.. include:: /referenceinfo/imports/software/ansys/ansys-license-restrictions.rst`

**Comment by Carldkennedy** [comment]
Github was having issues whilst I was trying to push commits earlier. Hence the duplications and reverts.

File: referenceinfo/imports/software/ansys/ansys-license-restrictions.rst [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+
+.. note::
```
Comment (jkwmoore) [inline referenceinfo/imports/software/ansys/ansys-license-restrictions.rst:3]:
```suggestion
.. caution:: 
```

Want it to be a bit more prominent in orange?

File: FAQs.rst [hunk 2]
```diff
@@ -707,3 +707,33 @@ again, no extra configuration is required by the end user.
...
+.. _ansys_license_restrictions:
+
+Are there any license restrictions for ANSYS?
+----------------------------------------------
+
+ANSYS users are subject to a limit of 400 concurrent cores, out of a total of 4500 available. There are no limitations on 
+the number of ANSYS applications users can open, but multi-core licenses are checked out per application, 
+with more than 4 cores used concurrently, equal to the number of cores in use above 4. 
+This restriction applies to individual usernames on all devices on campus, including personal machines and HPC, 
+and only for ANSYS multi-core licenses on the research license server.
```
Comment (jkwmoore) [inline FAQs.rst:720]:
```suggestion
ANSYS users are subject to a per user limit of 400 concurrent cores via a maximum check out limit of 400 ANSYS multi-core licenses. There are no limitations on 
the number of ANSYS applications users can open however:

* Multi-core licenses are checked out per application;
* when more than 4 cores are used concurrently;
* with the number required equal to the number of cores in use greater than 4. 

This restriction applies on an individual user basis across all applications and devices concurrently, including personal machines, managed desktop machines and the HPC clusters.
```

This seemed a bit clunky so I wanted to smooth it out a bit.

**Comment by Carldkennedy** [comment]
Thank you @jkwmoore

**Review by jkwmoore — APPROVED** [review]
LGTM - please merge at 13:00.

File: FAQs.rst [hunk 3]
```diff
@@ -707,3 +707,33 @@ again, no extra configuration is required by the end user.
...
+   ==============================================       ===========================
+   On a desktop open using 4 cores                      4  - 4 = 0
+   On another desktop using 6 cores                     6  - 4 = 2
+   A job on ShARC using 12 cores                        12 - 4 = 8
+   A job on Stanage using 20 cores                      20 - 4 = 16
+   Another job on Stanage using 30 cores                30 - 4 = 26     
+   **Total**                                            **52**
+   ==============================================       =========================== 
+
+In the above scenario a user is using many ANSYS applications at once and has a total of 52 multi-core licenses checked out.
```
[the remark this answers is not in the document]
Comment (Carldkennedy) [reply FAQs.rst]: Thanks @nikishe. Made the change

**Review by nikishe — APPROVED** [review]

**Comment by jkwmoore** [comment]
Please delete stale branches!

**Comment by Carldkennedy** [comment]
> Please delete stale branches!

Have done @jkwmoore

**Pull request merged**